config/Reader.alpha.xcconfig

```diff
@@ -0,0 +1,6 @@
+#include "Common.alpha.xcconfig"
+#include "Version.public.xcconfig"
```

Comment:
Something we can decide later on: Happy for Reader to ship with the same version as WordPress and Jetpack (simpler setup) or shall we use a dedicated version for it?